### DISCUSSION TITLE
Ack all button in alarm view/alarm counter in header out of sync

### DISCUSSION THREAD

**Discussion by testermax** [frangoteam/FUXA#896] [Q&A]
Hi,
First of all, great solution!
I`m happy to contribute to this, since I have 8+ years with angular/SCADA.

I have a question regarding the ack all button in alarm view.
When I use this, the alarm counter shown in the alarm bell (header) gets out of sync on what is shown in the alarm view.
Is this a bug? It seems like a bug, but I wanted to check before I report it.
I know I can figure this out by looking at the source, but if this is o know issue, it would save me the time.

Thanks.

**Comment by unocelli** [comment]
Hi, you are welcome.
About ack of alarm, have you also checked the Ack Mode ?
![image](https://github.com/frangoteam/FUXA/assets/48684550/3ebf5c3b-f69e-4042-b493-aee6a0fd1ec1)
